### ISSUE TITLE
Autobuild does not copy some static file changes

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/arcade#2266]
## System Info
Tip of `development`

### Actual behavior:
Sphinx autobuild:
1. detects the change in the CSS file and triggers rebuild
2. does not seem to copy the CSS file

### Expected behavior:
Sphinx autobuild:
1. detects the change in the CSS file and triggers rebuild
2. copies the CSS file

**Comment by pushfoo** [comment]
It looks like thumbnail generation also doesn't fire when those files are updated.

**Comment by einarf** [comment]
It this still a problem?

**Comment by Infamous003** [comment]
It is still a problem. Any changes made to the stylesheets cause the page to reload, but the styles aren't applied.
A painful fix is to do a fresh build, but that takes too much time.

**Comment by pushfoo** [comment]
> It this still a problem?

**TL;DR:** Our pre-3.0 temp fix should work until we test and deploy upstream's fix for their 10+ year bug

## The Quick Fix

> It is still a problem.

@Infamous003 I want you to try the temp fix:

1. `cd` into your Arcade repo's top-level directory
2. `touch .ENABLE_DEVMACHINE_SPHINX_STATIC_FIX`
3. Check if suddenly works

I apologize for not having this documented. Others seemed unbothered by this `_static` issue, so I made it opt-in only. Everyone who uses it might've forgotten it exists.

## Do I need to read the rest?

| I want "funny" | I want code details |
|------------------|-----------------------------|
| Continue reading | See bottom of comment |

Most of this comment is useful levity in the tradition of Arcade repo classics like:

* ["Reinvent the Rectangle"](https://github.com/pythonarcade/arcade/pull/2088)
* ["At the Mountains of Tech Debt"](https://github.com/pythonarcade/arcade/pull/2109)

...So without further ado, it's time for _another_  Python Arcade Story :tm:!

## Oops! All Forced Rebuilds!

> [!WARNING]
> Sit down and pour a `$BEVERAGE` of choice, because this starts 3.4 billion years ago in the [Sphinx](https://github.com/sphinx-doc/sphinx) repo.

### What happened?

1. Sphinx:
    1. had and fixed a bug (yay)
    2. but not in time for Arcade 3.0 (oh no)
2. We fixed our `_static` copy issues like everyone does:
    1. a helper script
    2. an ugly call in `conf.py`
3. It was [supposed to be temporary], so of course people didn't know about it

 
[supposed to be temporary]: https://en.wiktionary.org/wiki/there_is_nothing_more_permanent_than_a_temporary_solution

### In More Detail

| Event | Geological Era | Relevant Link |
|-------------------|-------|-------------|
| 10 yr Sphinx bug | [Archean (3.4 BYA)](https://en.wikipedia.org/wiki/Earliest_known_life_forms) | ["Files from `_static` are not copied automatically" (sphinx/#1810)](https://github.com/sphinx-doc/sphinx/issues/1810) |
| 3.0 + kludge | [Jan 25, 2025](https://pypi.org/project/arcade/3.0/) | ["Temp fix decade-long Sphinx bug breaking our CSS during partial rebuilds" (arcade/#2496)](https://github.com/pythonarcade/arcade/pull/2496) |
| Fix released(?) | Feb 11, 2025 | [GH actions locks fix as "resolved" / limited to collaborators](https://github.com/sphinx-doc/sphinx/pull/13236#event-16273003292) |


#### Our Temp Fix

1. [`util/sphinx_static_file_temp_fix.py`](https://github.com/pythonarcade/arcade/blob/development/util/sphinx_static_file_temp_fix.py)
2. [These lines:](https://github.com/pythonarcade/arcade/blob/3501dbd6f26819e639d85bbd1ef7d2e3390eb853/doc/conf.py#L125C1-L127C43):
```py
# Temp fix for Sphinx not copying static files  # pending: post-3.0 refactor
# Enable by creating a .ENABLE_DEVMACHINE_SPHINX_STATIC_FIX
run_util("sphinx_static_file_temp_fix.py")
```
